packages/collector/test/tracing/database/db2/test.js

```diff
@@ -852,7 +852,10 @@ mochaSuiteFn('tracing/db2', function () {
         .then(() =>
           testUtils.retry(() =>
             verifySpans(agentControls, controls, {
-              spanLength: 11,
+              // 11 queries splitted from the file
+              // 5 fs operations on top (ours + from db2 internally fs-extra)
+              // https://github.com/ibmdb/node-ibm_db/blob/fb25937524d74d25917e9aa67fb4737971317986/lib/odbc.js#L916
+              spanLength: 15,
```

Comment:
_nitpick:_ `numberOfSpans` might be a better name.

Reply:
Will rename.